src/pages/App.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import logo from './logo.svg';
+import '../styles/App.scss';
+import { FrontEndMentorHeader } from '../components/FrontEndMentor';
+import { Filter } from '../components/Filter';
+import { Roadmap } from '../components/Roadmap';
+import { ToolBar } from '../components/ToolBar';
```

Comment:
If you want, you can apply the same folder structure for the `/components` as we did for `types`. That way you can have all of your imports in one line and it reduces duplication. IMO it is cleaner to do it that way, but your choice :)

Reply:
Implemented!